docs/src/main/asciidoc/Getting_Started_on_OpenShift.adoc

```diff
@@ -0,0 +1,160 @@
+= Getting Started with WildFly on OpenShift
+
+This guide is designed to assist you through obtaining an OpenShift instance and installing WildFly, complete with a deployed application. It is not intended to cover all possible configurations, as links to more detailed documentation will be provided.
+
+=== Obtaining an OpenShift Instance
+There are many https://www.redhat.com/en/technologies/cloud-computing/openshift/try-it[options] available for running OpenShift. This guide will mention two.
+
+===== Code-ready-containers
+If you wish to run a local development environment, then https://developers.redhat.com/products/openshift-local/overview[CRC] is likely to be a good starting point.
+
+===== Developer sandbox
+The Red Hat OpenShift https://developers.redhat.com/developer-sandbox[Developer Sandbox] is a hosted environment, and is free to use for development purposes.
+
+This guide will focus on usage of the hosted developer sandbox environment. The sandbox should be provisioned from the page above before proceeding further.
+
+=== OpenShift
+In order to use OpenShift effectively, in addition to a running instance, you will need the OpenShift `web console`. To access this tool, from your OpenShift sandbox click the terminal icon (i.e. `>_`) at the top right of your sandbox portal. A terminal tab will pop up at the bottom of the page.
+
+It is also possible to use the OpenShift command line terminal locally (`oc` command). This may be obtained from your package manager or downloaded from your OpenShift sandbox (in your control panel, clicking on the question mark icon at the top right of the console presents a drop-down menu where `Command line tools` takes you to the latest version of `oc`).
+
+WARNING: If you have already installed `oc` on your machine, make sure that its version matches the version of the sandbox:
```

Comment:
Maybe we could also mention how to check the version of the sandbox, i.e., something like:

To check the version of the sandbox, click on the question mark icon at the top right of the web console and then click on `About` in the drop-down menu. The OpenShift version will be listed there.

Reply:
I've reworded the `WARNING`. Hopefully, it is clearer now :-) Thanks @fjuma! :pray: